assets/custom.js

```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
 function activate_dropdown() {
 	// body...
 	$(document).ready(function(){
-		$('.ui.selection.dropdown').click(function(){
-			$('.ui.dropdown').dropdown('show');
+		$('#project_filter').click(function(){
```

Comment:
and, you don't need to use `click` listener here as the code below `$('#project_filter').dropdown();` means element with id `project_filter` will have `dropdown` 'feature'.